src/content/Docs/deployments/console-api/deployment-lifecycle/index.md

```diff
@@ -0,0 +1,86 @@
+---
+categories: [Akash Console API]
+tags: ["API", "Workflows"]
+weight: 2
+title: "Deployment Lifecycle"
+linkTitle: "Deployment Lifecycle"
+---
+
+# Deployment Workflows
+
+This guide details the logical steps required to manage the full lifecycle of a deployment on Akash using the Console API.
+
+## The Deployment Lifecycle
+
+1.  **Certificate Creation**: Authenticate with providers.

```

Comment:
The certificate creation is now entirely optional and shouldn't be part of the deployment lifecycle as we use jwt under the hood. I suggest you remove it entirely and create an optional section for authentication at the bottom of the docs